### PR TITLE
[15.0][MIG] base_rest_datamodel: Migration to 15.0

### PR DESCRIPTION
**Pull request by yankinmax**
depends on:

* [x] #225
* [x] #222

### CODE REVIEW

**Comment by yankinmax** [comment]
@lmignon Can you pls take a look here?

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by lmignon — CHANGES_REQUESTED** [review]
@yankinmax Can you remove your last commit since all the required dependencies are now merged

**Comment by yankinmax** [comment]
> @yankinmax Can you remove your last commit since all the required dependencies are now merged

rebased

**Comment by yankinmax** [comment]
@simahawk can you pls take a look here also?

File: base_rest_datamodel/restapi.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2020 ACSONE SA/NV
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-
+# flake8: noqa
```
Comment (lmignon) [inline base_rest_datamodel/restapi.py]: @yankinmax Why this 'flake8: noqa` We must avoid global flake8 ignore

Comment (yankinmax) [reply base_rest_datamodel/restapi.py]: I understood. Actually I forgot, that I left it here. Thanks for pointing.

**Comment by yankinmax** [comment]
@lmignon @simahawk can it be approved now?

**Review by lmignon — APPROVED** [review]

**Pull request merged**